### DISCUSSION TITLE
I would love a screen showing pull requests across my organization

### DISCUSSION THREAD

**Discussion by amyAMeQ** [community/community#34709] [Pull Requests]
The 'Pull Requests' link at the top shows MY pull requests across all repositories.  I would LOVE one that showed PRs submitted by others in my organization across all the Repositories that I have access to.  Bonus points if you show how many reviews each one already has and an indicator if I've already reviewed it.

**Comment by jfullstackdev** [comment]
you have this although not exactly 

https://github.com/pulls

and if it is an Organization, it has a separate dashboard solely for all Org's repos, not only PRs, every commit is included

**Reply by amyAMeQ** [reply]
Ah!  I needed to change the filter to make it show everybody's.  I can bookmark the link without the author, and that will work just fine.  TY.

Our dashboard doesn't show PRs.  Not sure if the org. configured it that way, or if I'm just not seeing them.  We have to go to each repository and click on the Pull Requests link at the top. (Or click the PR request that was posted in slack)

**Comment by jfullstackdev** [comment]
I'm not very sure of your situation, I just have this right now 

![image](https://user-images.githubusercontent.com/47092464/193378182-98d35743-d995-4bde-8ce1-d221218b85c4.png)

even the commit in the branch is included at the dashboard

as you can see on the left pane, it's even listed

after the merged, you can see all of the details

![image](https://user-images.githubusercontent.com/47092464/193378311-328138a5-6055-49d8-a31c-c65c33077b90.png)

that behavior for Orgs is by default, I did not change anything in the setup
